style-guide/about-style-guide.md

```diff
@@ -0,0 +1,183 @@
+possible by deferring to a more comprehensive default guide, such as the Google
+Developer's Style Guide or the Microsoft Manual of Style. In this section,
+you should indicate which guide is your default guide and reference the source
+document and its version. Document any exceptions where your style guide differs
+from your default guide.
+
+Also, if your project already has existing brand guidelines, you can link to
+them or include them here.
+
+## About the "Glossary of preferred terms" section
+
+Every project has its own unique terms and buzzwords and this glossary can help
+you record your project’s preferred terms. This word list provides a list of
+specific terms that are either common word choice problems or which are unique
+to your project. As a word of caution, try to minimize jargon and buzzwords.
+Also, avoid creating too many project-specific terms. Where possible, use the
+terms that are standard for your industry to avoid confusion. Ideally, you can
+inherit or defer to term definitions from a more authoritative glossary source.
+
+This section includes a table to highlight or call out any terms that are unique
+or which conflict with your default style guide. For example, the Google
+Developer's Guide uses "open source" but our project prefers "open-source" when
+used as a modifier, as in "open-source software."
+
+## About the "Topic types and templates" section
+
+Depending on the type of project you’re creating, some of the Good Docs
+templates will be more relevant than others. In this section, you can link to
+the templates you have adopted for your project. You can also explain when
+contributors should use these templates.
+
+## About the "General writing tips" section
+
+This section is optional. You can link to your favorite general writing tips or
+add some of your own general guidelines.
+
+## About the "Accessible writing" section
+
+In this section, you can link to your preferred style guides about accessible
+writing or add some of your own general guidelines.
+
+## About the "Inclusive and bias-free writing" section
+
+In this section, you can link to your favorite resources about inclusive and
+bias-free writing or add some of your own general guidelines.
+
+## About the "Using linters" section
+
+This section is optional. [Linters](https://www.writethedocs.org/guide/tools/testing/#style-guide-checking-and-linting)
+are tools that analyze text content to flag common mistakes such as stylistic
+errors and suspicious constructs. An example of a linter would be a tool like
+Grammarly that processes your text and provides suggestions as you write to
+improve the spelling and grammatical structure of the text.
+
+Linters can be used during the editorial process within an editor and as a
+post-processing test of the content when the document is pushed into a revision
+management system as part of an automated continuous integration (CI) cycle.
```

Comment:
This is a very long sentence!

Reply:
My edit:
Linters can be used at two different points when creating documentation:
- **During the editorial process** - Typically used within a text editor, a
  linter can check for style consistency while composing documentation.
- **As a post-processing test** - When the content is pushed into a revision
  management system as part of an automated continuous integration (CI) cycle,
  a linter can evaluate text for style consistency.
- **As a post-processing test** - When the content is pushed into a revision
  management system as part of an automated continuous integration (CI) cycle,
  a linter can evaluate text for style consistency.

Reply:
I think you need to delete the third bullet there, but otherwise that looks fine to me :+1:

Reply:
That's odd that it got in there twice in my copy. It's correct in the version I pushed at least. Should be good now!